### ISSUE TITLE
Hello,  is it possible to track more than one inverter? I tried to run docker with deye-mqtt2 but in samrthome both inverter send to same variable

### ISSUE THREAD

**Issue by Jagu1967** [kbialek/deye-inverter-mqtt#75]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by kbialek** [comment]
Yes, it is possible. Read this comment https://github.com/kbialek/deye-inverter-mqtt/issues/46#issuecomment-1479541330

**Comment by Jagu1967** [comment]
thanks, perfect :-)
